docs/platform/howto/byoc/networking-security.md

```diff
@@ -0,0 +1,124 @@
+allows the Aiven Management plane to launch services within your private cloud.
+
+### Traffic to the bastion
+
+Aiven Management plane traffic to your BYOC environment originates from a set of redundant
+gateways with fixed IPs. Therefore, the bastion subnet needs to whitelist four static IPs
+on relevant ports.
+
+| Source                 | Destination    | Description                                                                                                                                   |
+| ---------------------- | ---------------| --------------------------------------------------------------------------------------------------------------------------------------------- |
+| Aiven Management plane | Bastion subnet | Aiven uses SSH from the Aiven Management plane to the bastion service node for troubleshooting, setup, or configuration tasks on the bastion. |
+| Aiven Management plane | Bastion subnet | Proxy service is used to tunnel the traffic from the Aiven Management plane to workload nodes.                                                |
+| Aiven Management plane | Bastion subnet | Aiven Management plane accesses the bastion service node to retrieve status information and perform routine operations.                       |
+
+### Traffic from the bastion
+
+| Source         | Destination      | Description                                                                                                                             |
+| -------------- | -----------------| --------------------------------------------------------------------------------------------------------------------------------------- |
+| Bastion subnet | Aiven Management | Communication channel with the Aiven Management plane used to collect metrics and logs from the bastion and workload nodes              |
+| Bastion subnet | Aiven Management | Required for the bastion and workload nodes to make calls to the Aiven Management plane                                                 |
+| Bastion subnet | AWS CloudFront   | Used for downloading RPM packages for bastion nodes setup                                                                               |
+| Bastion subnet | Workload subnet  | Aiven uses SSH from the bastion subnet to the workload subnet for troubleshooting, setup, or configuration tasks on the workload nodes. |
+| Bastion subnet | Workload subnet  | Aiven Management plane accesses the workload nodes via the bastion node to retrieve status information and perform routine operations.  |
+| Bastion subnet | DNS & NTP        | Destination dependant on cloud provider                                                                                                 |
+
+:::note
+Aiven Management and CloudFront IP ranges do occasionally change and are considered dynamic
+for the purposes of firewall policies. To that end, an egress of `0.0.0.0/0` is generally
+required to accommodate traffic to these destinations.
```

Comment:
Just a suggestion, maybe we can simplify this a bit: 

Aiven Management and CloudFront IP ranges occasionally change and are considered dynamic for firewall policies. To accommodate traffic to these destinations, an egress of 0.0.0.0/0 is generally required. 